plugins.xml

```diff
@@ -4513,6 +4513,32 @@
 			<institution>University of Pittsburgh</institution>
 			<email>uls-systemsdevelopment@pitt.edu</email>
 		</maintainer>
+		<release date="2020-06-29" version="1.0.0.0" md5="789e6f18c743359bc548c5455efebd5f">
```

Comment:
This change is "noisy".  By reordering the releases, we see version 1.0.0.0 as a removal and an addition and 1.0.1.0 as a change.

To make it easier on maintainers, this kind of cleanup should be submitted in a different pull request which describes the intent of the cleanup, or simply forgo the cleanup in preference of readability of the merge.  To add this cleanup into a PR which is only described as updating "Email Issue TOC 3.2" adds unnecessary cognitive load for whoever is merging it.